service/orchestrator/catalogs.json

```diff
@@ -0,0 +1,31 @@
+[
```

Comment:
More meaningful catalog once we are done. Maybe something similar to the old requirements.json.

Reply:
I have added a few more fields and more descriptive names, but this is not what you mean I assume?

Reply:
In the old file (see https://github.com/clouditor/clouditor/blob/main/service/orchestrator/requirements.json) we had two exemplary requirements that were connected to real metrics. So I would assume that we would either create our "own" catalog with more or less "fake" controls or maybe choose an appropriate EUCS control that is at least connected to a real metric such as TransportEncryptionEnabled.